requirements.txt

```diff
@@ -57,7 +57,9 @@ google-cloud-pubsub==2.16.0
 google-cloud-run==0.5.0
 google-cloud-storage==2.5.0
 kubernetes==25.3.0
+langchain>=0.0.222
```

Comment:
You can update it to `langchain>=0.0.222; python_version >= '3.8'`

Reply:
Updated